config/services.yaml

```diff
@@ -30,5 +30,4 @@ services:
 
     Surfnet\AzureMfa\Application\Service\AuthenticationHelper:
         arguments:
-            $regex: '%ra_issuer_entity_id_regex%'
-            $authenticationService: '@Surfnet\GsspBundle\Service\AuthenticationService'
\ No newline at end of file
+            $regex: '%ra_issuer_entity_id_regex%'
```

Comment:
Consider not using a regex for KISS and DID

Reply:
Not something to fix here in my opinion. And for now this is what PM and I came up with.